charset.c

```diff
@@ -519,8 +520,8 @@ int mutt_convert_string (char **ps, const char *from, const char *to, int flags)
 
 
 /*
- * FGETCONV stuff for converting a file while reading it
- * Used in sendlib.c for converting from mutt's Charset
+ * FGETCONV stuff for converting a file while reading it Used in sendlib.c for
+ * converting from mutt's Charset
  */
```

Comment:
Sorry I don't know straight away, we'd need to test but I don't have a computer at hand at the moment :/